### ISSUE TITLE
Severity colors not showing

### ISSUE THREAD

**Issue by quonic** [screepers/screeps_console#5]
If I put a character in front of the font tag, Like a double ||, then no color is applied to the font.

See example output: http://i.imgur.com/vBwXz67.png

Thanks!

**Comment by tedivm** [comment]
The code that does this is looking at the root tag to get the severity. By putting code outside of the tag you're making it so it is no longer root.

To make that work you should wrap the entire message in a `log` tag and put the severity in that (`<log severity=3>Message <font color=purple>with color!</font></log>`).

**Comment by quonic** [comment]
I've tried the following formats:
`<log severity=3>message</log>`
`<log severity=3><font color=purple>Message</font></log>`

Neither work. I have a leading and trailing "║" which I think might be the cause. If I remove it, it still doesn't color.

**Comment by tedivm** [comment]
Can you show me a complete console raw output with all of the tags in place? Something I can put straight into `console.log()` and test myself.

**Comment by quonic** [comment]
Raw output from one console.log

`<span style="line-height: 1;">╔═════════════════════════════════════════════ Logs ══════════════════════════════════════════════╗<br>║<log severity="0">Spawn1 Respawning E41S7Redistributor1</log>                                                            ║<br>║<log severity="0">Spawn2 Respawning E41S6Scout2RemoteTruck2</log>                                                        ║<br>║<log severity="1">[room E41S9] Controller Ticks left:4500</log>                                                          ║<br>║<log severity="0">Spawn3 Respawning E41S8Redistributor1</log>                                                            ║<br>║<log severity="5">Tick: 12289333  CPU OVERRUN: 74.76  Bucket:10000</log>                                                 ║<br>║<log severity="0">Trying to create E41S7ReserverScout1</log>                                                             ║<br>║<log severity="0">Trying to create E41S7ReserverScout1</log>                                                             ║<br>║<log severity="5">Tick: 12289356  CPU OVERRUN: 50.31  Bucket:10000</log>                                                 ║<br>╚═ Tick: 12289375 ════════════════════════════════════════════════════════════════════════════════╝<br></span>`

**Comment by quonic** [comment]
So the cause, that we talked over on slack, was that I was outputting multiple lines with one console.log call. Changing my code to output to multiple console.log calls with each call wrapped in <log> tags fixed it. Not Ideal, but it does fix it. I did find that if I wrap the whole thing in a <log> tag that it doesn't color it either. I suspect that the <br>'s are to blame some how, but I don't fully understand how it does the parsing of the console.log's.

Thanks for the help tedivm!
